packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -80,7 +80,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
     IEventProvider<IDeltaManagerInternalEvents> {
     public readonly connectionManager: TConnectionManager;
 
-    public get active(): boolean { return this._active(); }
+    public get active(): boolean { return this._active() && this.connectionManager.connectionMode === "write"; }
```

Comment:
I do not think it changes anything, as Container.activeConnection() already checks for "write"
I was hoping we can move logic completely here and avoid _active() callback, but I do not think it's possible, as only Container knows if client is in quorum, so likely not much we can do to improve it.